### PR TITLE
Add Discord badge link

### PR DESCRIPTION
**Pull request by WMCB-Tech**
I guess discord is used by most people, and basically some people don't use gitter

i might add this badge to other repos as well

### CODE REVIEW

**Comment by agnostic-apollo** [comment]
Sure.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Build status](https://github.com/termux/termux-app/workflows/Build/badge.svg)](https://github.com/termux/termux-app/actions)
 [![Testing status](https://github.com/termux/termux-app/workflows/Unit%20tests/badge.svg)](https://github.com/termux/termux-app/actions)
 [![Join the chat at https://gitter.im/termux/termux](https://badges.gitter.im/termux/termux.svg)](https://gitter.im/termux/termux)
+[![Join the Termux discord server](https://img.shields.io/discord/591914197219016707.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/HXpF69X)
```
Comment (leapofazzam123) [inline README.md:6]: It still using the old blurple, plus the discord icon has been updated from the old one. You may want to use this instead:
https://img.shields.io/discord/591914197219016707.svg?label=&logo=discord&logoColor=ffffff&color=5865F2&style=for-the-badge

[Join the Termux discord server](https://discord.gg/HXpF69X)

Comment (WMCB-Tech) [reply README.md:6]: will do

**Comment by rmjspaz7284** [comment]
> I guess discord is used by most people, and basically some people don't use gitter
> 
> i might add this badge to other repos as well

**Pull request merged**